frontend/lib/identification/verification_workflow/verification_qr_code_processor.dart

```diff
@@ -42,6 +70,12 @@ void _assertConsistentDynamicVerifyCode(DynamicVerifyCode verifyCode) {
   }
 }
 
+void _assertConsistentStaticVerifyCode(StaticVerifyCode verifyCode) {
+  if (!verifyCode.hasPepper()) {
+    throw QrCodeFieldMissingException("hashSecretBase64");
```

Comment:
```suggestion
    throw QrCodeFieldMissingException("pepper");
```

Reply:
So is it called pepper when there is no `totpSecret` and `hashSecretBase64` when there is a `totpSecret`?

Reply:
no, it is always called papper :)